src/store/eventing/fetch.ts

```diff
@@ -0,0 +1,11 @@
+import eventManager from '../../common/EventManager';
+import { AppState } from '../types';
+import { AsyncAction } from './types';
+import { Event } from '../../@types';
+
+const handleFetchErrorEvents = (prevState: AppState, nextState: AppState, action: AsyncAction): void => {
```

Comment:
Let's use `export const` to avoid having to track a separate export literal for the file.